### PR TITLE
Feat : added badge component

### PR DESCRIPTION
**Pull request by poornima-alugubelly**

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]

File: components/Badge/Badge.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  display: flex;
+
+  align-items: center;
+  justify-content: center;
+}
+
+.badge-top {
+  top: -11px;
+  right: -13px;
+}
```
Comment (mimansha-swarup) [inline components/Badge/Badge.css:27]: i guess giving absolute value will not work for every cases
if its working then alright 👍

**Review by mimansha-swarup — COMMENTED** [review]
just see that comment 
except that everything looks good to go

**Review by manojasarsa — APPROVED** [review]
Good to go!

File: components/Badge/Badge.css (hunk 1 above)
Comment (poornima-alugubelly) [reply components/Badge/Badge.css:27]: If text is not multiline it works perfectly if its multiline its  kind of off but okay the main reason for using fixed values was when I used %s I was not able to place the badge at the top it kind of stays on the text even when I use top:1%

**Review by MarkVed17 — APPROVED** [review]

File: components/Badge/Badge.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  color: white;
+}
+
+.badge-green {
+  background-color: green;
+  color: white;
+}
+
+.badge-yellow {
+  background-color: rgb(245, 210, 12);
```
Comment (KoushikKintali) [inline components/Badge/Badge.css:55]: derive color from variable

**Comment by sakshamAK** [comment]
Great work!! Your landing page is awesome😍
Just a suggestion for future reference, if percentage doesn't work good, try em units

File: components/Badge/Badge.css (hunk 2 above)
Comment (poornima-alugubelly) [reply components/Badge/Badge.css:55]: Hey Koushik thanks for the review!
 that is the only place that color will be used hence I left it that way

**Pull request merged**